article/app/AppLoader.scala

```diff
@@ -20,14 +20,17 @@ import play.api.routing.Router
 import play.api._
 import services.{NewspaperBooksAndSectionsAutoRefresh, OphanApi}
 import router.Routes
+import conf.Configuration
 
 class AppLoader extends FrontendApplicationLoader {
   override def buildComponents(context: Context): FrontendComponents = new BuiltInComponentsFromContext(context) with AppComponents
 }
 
 trait AppComponents extends FrontendComponents with ArticleControllers {
 
-  lazy val capiHttpClient: HttpClient = wire[CapiHttpClient]
+  lazy val _isDevInfra = Configuration.environment.stage.equalsIgnoreCase("DEVINFRA")
+  lazy val capiHttpClient: HttpClient = if (_isDevInfra) wire[MockHttpClient] else wire[CapiHttpClient]
```

Comment:
I want to run the actual app (as opposed to tests), without remote services, so it can be location agnostic.

we're going to run the app on an instance and point Gareth's screenshots and mike's analytics stuff to it. This is a lot easier if we don't have to worry about configuring for location of remote services (ie just reuse devinfra)

As a side effect this also makes it possible for people external to the Guardian to contribute  changes more easily, a point which was on one of the big bullet point list of nice things to do.